index.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+require_once 'AlbumsDAO.php';
+require_once 'ArtistsDAO.php';
+
+// CREATE ARRAYS OF ALBUMS AND ARTISTS //
+$albumsDAO = new AlbumsDAO();
+$albums = $albumsDAO->fetchAll();
+
+$artistsDAO = new ArtistsDAO();
+$artists = $artistsDAO->fetchAll();
+
+function intToBritDate(string $intDate): string {
+    $timestamp = strtotime($intDate);
+    return date("d.m.Y", $timestamp);
+}
+
+// ADD ARTISTS TO EACH ALBUM OBJECT //
+foreach ($albums as $album) {
+    $album_id = $album->getId();
+    $albumartists = [];
+    foreach ($artists as $artist) {
+        if ($album_id == $artist['album_id']) {
+            $albumartists[] = $artist['artist'];
+        }
+    }
+    $albumartists = implode(", " , $albumartists);
+    $album->setArtists($albumartists);
+}
```

Comment:
This seems like like something that should be moved into a function as well - always a good idea to put things into functions so you can reuse them easily in future

Reply:
This would also need a unit test